django_ipam/api/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.conf.urls import url
+
+from . import views
+
+urlpatterns = [
+    url(r'^iprequest/$', views.iprequest, name='iprequest'),
```

Comment:
please add the `id` in the URL and use that instead of the querystring.

Should be `/api/v1/subnet/<id>/iprequest/`